IHP/Controller/Param.hs

```diff
@@ -399,6 +399,24 @@ instance ParamReader UTCTime where
     readParameterJSON (Aeson.String string) = readParameter (cs string)
     readParameterJSON _ = Left "ParamReader UTCTime: Expected String"
 
+-- | Accepts values such as @2020-11-08T12:03:35Z@ or @2020-11-08@
+instance ParamReader LocalTime where
```

Comment:
Could you add a test for this? [Likely we can just copy the UTCTime tests](https://github.com/digitallyinduced/ihp/blob/master/Test/Controller/ParamSpec.hs#L249)

Reply:
done